src/jekyll/Gemfile

```diff
@@ -0,0 +1,28 @@
+source 'https://rubygems.org'
```

Comment:
Location of jekyll related file should be **src/homepage/jekyll**

Here, with your proposal, we don't know what this is used for.